config/initializers/devise.rb

```diff
@@ -0,0 +1,274 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '148d20ff8214af713ea5e1b4a2541a69ce7bc2aed76bcf3fe17148d655e030a857b1cb5b1e1494a8c14136401a8800006f92f1e0663844676767d2e2389e30eb'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Will probably want to change this sooner rather than later, so you don't forget about it.